qcodes/instrument_drivers/Harvard/Decadac.py

```diff
@@ -115,8 +162,8 @@ def _setvoltage(self, voltage, channel):
             try:
                 self.visa_handle.read()
             except UnicodeDecodeError:
-                logging.warning(" Decadac returned nothing and did nothing. " +
-                                "Please re-run the command")
+                log.warning(" Decadac returned nothing and did nothing. " +
+                            "Please re-run the command")
                 pass
 
         if self.ramp_state:
```

Comment:
ok reload fixes this but I'm kind of surprised it can even make that error message :(